### PR TITLE
[CHORE] Update Login Settings Menu to be in line with Game Options implementation

### PR DESCRIPTION
**Pull request by eliasSFL**
# Description

- Moved Font Settings to a separate file
- Update Login Settings to be in line with Game Options implementation

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

# Checklist:

- [ ] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [ ] Screenshot if it includes any UI changes
- [ ] I have read the contributing guidelines and agree to the T&Cs
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by eliasSFL** [comment]
This PR will cause conflicts with #3853 so have to update the other PR whichever PR merges in first

File: src/features/auth/components/LoginSettings.tsx [hunk 1]
```diff
@@ -1,53 +1,64 @@
...
+    setMenu("login");
   };
-  const Content = () => {
-    if (showFonts) {
-      return <AppearanceSettings />;
-    }
 
-    if (showLanguage) {
-      return <LanguageSwitcher />;
+  const Content = () => {
```
Comment (spencerdezartsmith) [inline src/features/auth/components/LoginSettings.tsx:28]: Hey @eliasSFL, I know you didn't create this component but now is a good time to refactor this as creating a "component" inside another component is generally bad practice. You can read about it [here](https://dev.to/sanjampreetsingh/nested-components-a-bad-practice-reactjs-1hbf). 

In this case this code inside of `Content` can just be moved inline in the JSX return like the below example:

```
{currentMenu !== "login" && <code>}
{currentMenu === "login" && <code>}
```

**Review by spencerdezartsmith — CHANGES_REQUESTED** [review]
This is generally good to go but I mentioned one refactor opportunity 🙏🏼

**Comment by eliasSFL** [comment]
@spencerdezartsmith I have updated the code with your requested changes

**Review by spencerdezartsmith — APPROVED** [review]
👏🏼

**Pull request merged**